pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -312,10 +314,13 @@ func (m *kubeGenericRuntimeManager) makeMounts(opts *kubecontainer.RunContainerO
 				utilruntime.HandleError(fmt.Errorf("unable to set termination-log file permissions %q: %v", containerLogPath, err))
 			}
 
+			// Volume Mounts fail on Windows if it is not of the form C:/
```

Comment:
Why don't we need to check the runtime here?

Reply:
There's no need. Which mounts to make on the pod are already decided by this point, in kubelet_pods.go to be more exact. In there, it is decided which mounts the pod will have, including if ``/etc/hosts`` file and the ``termination-log`` file are to be mounted.